src/catalogue/category/property/addPropertyMigrationDialog.component.test.tsx

```diff
@@ -0,0 +1,683 @@
+    });
+
+    await user.click(
+      screen.getByRole('checkbox', {
+        name: 'Confirm understanding and proceed checkbox',
+      })
+    );
+    await user.click(screen.getByRole('button', { name: 'Save' }));
+
+    const duplicateName = screen.getByText(
+      'Duplicate property name. Please change the name.'
+    );
+
+    expect(duplicateName).toBeInTheDocument();
+
+    await modifyValues({
+      name: 'Axis2',
+    });
+
+    const duplicateName2 = screen.queryByText(
+      'Duplicate property name. Please change the name.'
+    );
+
+    expect(duplicateName2).not.toBeInTheDocument();
+  });
+
+  it('display error when the allowed values list is empty and clears when a new list value is added', async () => {
+    createView();
+    await modifyValues({
+      name: 'Axis',
+      type: 'Text',
+      unit: 'millimeters',
+      mandatory: 'false',
+      allowed_values: {
+        type: 'list',
+        values: [],
+      },
+    });
+
+    await user.click(
+      screen.getByRole('checkbox', {
+        name: 'Confirm understanding and proceed checkbox',
+      })
+    );
+    await user.click(screen.getByRole('button', { name: 'Save' }));
+
+    const allowedValuesError = screen.getByText(
+      'Please create a valid list item.'
+    );
+
+    expect(allowedValuesError).toBeInTheDocument();
+
+    await modifyValues({
+      allowed_values: {
+        type: 'list',
+        values: [''],
+      },
+    });
+
+    const allowedValuesError2 = screen.queryByText(
+      'Please create a valid list item.'
+    );
+
+    expect(allowedValuesError2).not.toBeInTheDocument();
+  });
+
+  it('display error when the allowed values list is empty and clears when allowed values is changed to any', async () => {
+    createView();
+    await modifyValues({
+      name: 'Axis',
+      type: 'Text',
+      unit: 'millimeters',
+      mandatory: 'false',
+      allowed_values: {
+        type: 'list',
+        values: [],
+      },
+    });
+
+    await user.click(
+      screen.getByRole('checkbox', {
+        name: 'Confirm understanding and proceed checkbox',
+      })
+    );
+    await user.click(screen.getByRole('button', { name: 'Save' }));
+
+    const allowedValuesError = screen.getByText(
+      'Please create a valid list item.'
+    );
+
+    expect(allowedValuesError).toBeInTheDocument();
+
+    const allowedValues = screen.getByLabelText('Select Allowed values *');
+
+    await user.click(allowedValues);
+    const allowedValuesDropdown = screen.getByRole('listbox', {
+      name: 'Select Allowed values',
+    });
+    await user.click(
+      within(allowedValuesDropdown).getByRole('option', {
+        name: 'Any',
+      })
+    );
+    const allowedValuesError2 = screen.queryByText(
+      'Please create a valid list item.'
+    );
+
+    expect(allowedValuesError2).not.toBeInTheDocument();
+  });
+
+  it('displays error for duplicate allowed values and clears when list value is deleted (type number)', async () => {
```

Comment:
```suggestion
  it('displays error for duplicate allowed values and clears when list value is deleted', async () => {
```
There are no other types tested here, so perhaps the type isn't needed as its the same logic presumably.